app/src/main/java/com/codesoom/controllers/ProductController.java

```diff
@@ -0,0 +1,90 @@
+package com.codesoom.controllers;
+
+import com.codesoom.application.ProductService;
+import com.codesoom.domain.Product;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * 상품에 대한 HTTP 요청의 처리를 한다.
+ */
```

Comment:
JavaDoc은 문장을 어법에 맞게 잘 적어주는 것이 중요합니다. JavaDoc은 그냥 주석이 아니라 빌드를 통해 하나의 정적 사이트로 만드는 기능도 갖고 있거든요.

```suggestion
/**
 * 상품에 대한 HTTP 요청의 처리를 담당한다.
 */
```